### PR TITLE
abi-stablity spanish version

### PR DESCRIPTION
**Pull request by cristiandelcidd**
Support for node Spanish documentation

### CODE REVIEW

**Comment by nschonni** [comment]
Thanks, can you take a look at the formatting issues
```
   18:539-18:561  warning  Use the trailing `[]` on reference links  no-shortcut-reference-link  remark-lint
    31:1-31:210  warning  Don’t pad `strong` with inner spaces      no-inline-padding           remark-lint
    31:16-31:21  warning  Don’t pad `emphasis` with inner spaces    no-inline-padding           remark-lint
```

File: locale/es/docs/guides/abi-stability.md [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
 
-To these ends Node.js has introduced N-API in version 8.6.0 and marked it as a stable component of the project as of Node.js 8.12.0. The API is defined in the headers [`node_api.h`][] and [`node_api_types.h`][], and provides a forward- compatibility guarantee that crosses the Node.js major version boundary. The guarantee can be stated as follows:
+Con estos fines, Node.js ha introducido N-API en la versión 8.6.0 y lo ha marcado como un componente estable del proyecto a partir de Node.js 8.12.0. La API se define en los encabezados [`node_api.h`] [] y [` node_api_types.h`] [], y proporciona una garantía de compatibilidad con versiones posteriores que cruza el límite de la versión principal de Node.js. La garantía puede expresarse de la siguiente manera:
```
Comment (nschonni) [inline locale/es/docs/guides/abi-stability.md]:
```suggestion
Con estos fines, Node.js ha introducido N-API en la versión 8.6.0 y lo ha marcado como un componente estable del proyecto a partir de Node.js 8.12.0. La API se define en los encabezados [`node_api.h`][] y [` node_api_types.h`][], y proporciona una garantía de compatibilidad con versiones posteriores que cruza el límite de la versión principal de Node.js. La garantía puede expresarse de la siguiente manera:
```

File: locale/es/docs/guides/abi-stability.md [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
 
-**A given version *n* of N-API will be available in the major version of Node.js in which it was published, and in all subsequent versions of Node.js, including subsequent major versions.**
+** Una versión * n * dada de N-API estará disponible en la versión principal de Node.js en la que se publicó y en todas las versiones posteriores de Node.js, incluidas las versiones principales posteriores. **
```
Comment (nschonni) [inline locale/es/docs/guides/abi-stability.md]:
```suggestion
**Una versión *n* dada de N-API estará disponible en la versión principal de Node.js en la que se publicó y en todas las versiones posteriores de Node.js, incluidas las versiones principales posteriores.**
```

**Review by nschonni — CHANGES_REQUESTED** [review]

**Comment by nschonni** [comment]
Ping @nodejs/nodejs-es

File: locale/es/docs/guides/abi-stability.md [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
+## Estabilidad ABI en Node.js
+Node.js proporciona archivos de encabezado mantenidos por varios equipos independientes. Por ejemplo, el equipo de Node.js mantiene archivos de encabezado como `node.h` y` node_buffer.h`. `v8.h` es mantenido por el equipo de V8, que, aunque en estrecha cooperación con el equipo de Node.js, es independiente y tiene su propio horario y prioridades. Por lo tanto, el equipo de Node.js solo tiene control parcial sobre los cambios que se introducen en los encabezados que proporciona el proyecto. Como resultado, el proyecto Node.js adoptó [versiones semánticas] (https://semver.org/). Esto asegura que las API proporcionadas por el proyecto darán como resultado una ABI estable para todas las versiones menores y de parche de Node.js lanzadas dentro de una versión principal. En la práctica, esto significa que el proyecto Node.js se ha comprometido a garantizar que un complemento nativo de Node.js compilado con una versión principal determinada de Node.js se cargue correctamente cuando lo cargue cualquier versión menor o parche de Node.js dentro de la principal. versión contra la que se compiló.
```
Comment (nschonni) [inline locale/es/docs/guides/abi-stability.md:18]:
```suggestion
Node.js proporciona archivos de encabezado mantenidos por varios equipos independientes. Por ejemplo, el equipo de Node.js mantiene archivos de encabezado como `node.h` y` node_buffer.h`. `v8.h` es mantenido por el equipo de V8, que, aunque en estrecha cooperación con el equipo de Node.js, es independiente y tiene su propio horario y prioridades. Por lo tanto, el equipo de Node.js solo tiene control parcial sobre los cambios que se introducen en los encabezados que proporciona el proyecto. Como resultado, el proyecto Node.js adoptó [versiones semánticas](https://semver.org/). Esto asegura que las API proporcionadas por el proyecto darán como resultado una ABI estable para todas las versiones menores y de parche de Node.js lanzadas dentro de una versión principal. En la práctica, esto significa que el proyecto Node.js se ha comprometido a garantizar que un complemento nativo de Node.js compilado con una versión principal determinada de Node.js se cargue correctamente cuando lo cargue cualquier versión menor o parche de Node.js dentro de la principal. versión contra la que se compiló.
```

**Review by wardaks24 — APPROVED** [review]

**Pull request closed without merging**